apps/src/code-studio/progress.js

```diff
@@ -3,42 +3,49 @@ import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
-import { createStore } from 'redux';
+import { createStore } from '@cdo/apps/redux';
```

Comment:
Use the createStore wrapper that we have in apps, so that we get some middleware (like a logger when you append `?enableExperiments=reduxLogger`.